cobbler.spec

```diff
@@ -229,6 +230,7 @@ Recommends:     syslinux
 Recommends:     %{grub2_x64_efi_pkg}
 Recommends:     %{grub2_ia32_efi_pkg}
 Recommends:     logrotate
+Recommends:     %{py3_module_librepo}
```

Comment:
Please just use `python%{python3_pkgversion}-librepo` here, we don't have alternate names to worry about for this module.

Reply:
I've replaced %{py3_module_librepo} with python%{python3_pkgversion}-librepo.